### PR TITLE
Check if container already exits

### PR DESCRIPTION
**Pull request by ludojmj**
The container may have already been created during a previous release.
Hence the script "create-container.ps1" must first check if the container exists in the storage account to avoid throwing an exception and therefore interrupt a new release process.

### CODE REVIEW

**Comment by carlosjdelgado** [comment]
Thanks @ludojmj !

I think this should be configurable inside the task, a checkbox that defines if task will be skipped when container exists for cover all needs.

Could you implement it? It should not be very complex.

**Comment by ludojmj** [comment]
Hello @carlosjdelgado 
I hope this implementation is what you were waiting for.

File: AzureStorageContainer/task.json [hunk 1]
```diff
@@ -57,7 +57,14 @@
...
         "Off": "No read access"
       }
-
+    },
+    {
+      "name": "ContinueOnExistence",
+      "type": "boolean",
+      "label": "Continue on container existence",
+      "defaultValue": false,
+      "required": false,
```
Comment (carlosjdelgado) [inline AzureStorageContainer/task.json]: Could you set this as required like in the PS script?

**Review by carlosjdelgado — COMMENTED** [review]
It seems correct, but I need some time for test the extensión before publish It. I hope during week. 

Thanks for contributing!

**Comment by ludojmj** [comment]
Done.

**Pull request merged**